receptorctl/receptorctl/cli.py

```diff
@@ -187,18 +187,45 @@ def ping(ctx, node, count, delay):
 @click.pass_context
 def reload(ctx):
     rc = get_rc(ctx)
-    results = rc.simple_command(f"reload")
+    results = rc.simple_command("reload")
     if "Success" in results and results["Success"]:
-        print(f"Reload successful")
+        print("Reload successful")
     else:
         print(f"Error: {results['Error']}")
         if "ERRORCODE 3" in results['Error']:
+            # config file was fine, but some changes were non-reloadable
             sys.exit(3)
         elif "ERRORCODE 4" in results['Error']:
+            # problem loading the config file, probably a syntax error
             sys.exit(4)
         else:
+            # problem loading the config file, probably a syntax error
```

Comment:
Should these changes be in this PR or a separate one? It looks like its reload related and not purge related.

Reply:
yeah not related, I just wanted to tidy up the code.